docs/api/ConnectionResumptionTicketValidationComplete.md

```diff
@@ -0,0 +1,43 @@
+ConnectionResumptionTicketValidationComplete function
+======
+
+Uses the QUIC (server) handle to complete resumption ticket validation. This must be called after server app handles ticket validation and then return QUIC_STATUS_PENDING.
```

Comment:
Does it actually need to be called before the app returns from the callback (with `QUIC_STATUS_PENDING`) or can it be called in the callback?

Reply:
And to be clear, I think it should be callable in the callback.

Reply:
Crypto status changes to TicketValidationPending==True after callback returns QUIC_STATUS_PENDING.
Calling this function in the callback could be indeterministic behavior.
do nothing, no error or complete

```
    if (!Crypto->TicketValidationPending || Crypto->TicketValidationRejecting) {
        return;
    }
```

Reply:
```c
        QuicTraceLogConnVerbose(
            IndicateResumed,
            Connection,
            "Indicating QUIC_CONNECTION_EVENT_RESUMED");
        Status = QuicConnIndicateEvent(Connection, &Event);

        if (Status == QUIC_STATUS_SUCCESS) {
            QuicTraceEvent(
                ConnServerResumeTicket,
                "[conn][%p] Server app accepted resumption ticket",
                Connection);
            ResumptionAccepted = TRUE;
        } else if (Status == QUIC_STATUS_PENDING) {
            QuicTraceEvent(
                ConnServerResumeTicket,
                "[conn][%p] Server app asynchronously validating resumption ticket",
                Connection);
            Connection->Crypto.TicketValidationPending = TRUE;
            ResumptionAccepted = TRUE;
        }
```